nltepop.cc

```diff
@@ -921,7 +931,8 @@ void solve_nlte_pops_element(const int element, const int modelgridindex, const
   // calculate the normalisation factors and apply them to the matrix
   // columns and balance vector elements
   gsl_vector *pop_norm_factor_vec = gsl_vector_calloc(nlte_dimension);
-  nltepop_matrix_normalise(modelgridindex, element, rate_matrix, pop_norm_factor_vec);
+  gsl_vector_set_all(pop_norm_factor_vec, 1.0);
+  // nltepop_matrix_normalise(modelgridindex, element, rate_matrix, pop_norm_factor_vec);
```

Comment:
This does change the outputs for cells with low densities. It might just be that both methods describe the cells well in this test model.

I think it would be a good idea to put some cells in these tests which have extreme densities/compositions as this would allow us to test the ability of functions to cope with grid cells in 3D models.